src/common/options.cc

```diff
@@ -6287,6 +6287,13 @@ std::vector<Option> get_rgw_options() {
     .set_default(1_M)
     .set_description("Send copy-object progress info after these many bytes"),
 
+    Option("rgw_copy_verify_object", Option::TYPE_BOOL, Option::LEVEL_ADVANCED)
+    .set_default(false)
+    .set_description("Verify if the object copied is identical to its source")
+    .set_long_description(
+        "If true, this option computes the MD5 checksum of the data which is written at the"
+	"destination and checks if it is identical to the ETAG stored in the source."),
```

Comment:
can we find a better name for this? i worry that 'copy' will be confused with the CopyObj operation. how about something like `rgw_fetch_remote_obj_integrity` or `rgw_sync_obj_integrity`? the description should mention that it applies to objects fetched from a remote over http, including multisite sync